pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -123,9 +125,18 @@ func (admitter *VMICreateAdmitter) Admit(ar *admissionv1.AdmissionReview) *admis
 
 	reviewResponse := admissionv1.AdmissionResponse{}
 	reviewResponse.Allowed = true
+	reviewResponse.Warnings = append(reviewResponse.Warnings, warnDeprecatedAPIs(vmi)...)
```

Comment:
Please elaborate in the commit message when the warning is produced and where is it expected to be seen.

(e.g. is it once per creation? Per update? Per patch?)

Reply:
This is the VM create admitter so this code runs on VM creation

Reply:
Looking at this again, its not enough its necessary to do the same at the VM addmitter to raise warning on VirtualMachine (not VMI) creation